emission/core/wrapper/common_place.py

```diff
@@ -0,0 +1,16 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonPlace(ecwb.WrapperBase):
+    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
+             "successors" : ecwb.WrapperBase.Access.WORM, # set of CommonPlaces connected to this location
+             "user_id" : ecwb.WrapperBase.Access.WORM,
+             "common_place_id" : ecwb.WrapperBase.Access.WORM
```

Comment:
why not just use _id? The good thing about _id is that mongodb will autopopulate it. How were you planning on generating a unique ID that would be better than mongo's version?